packages/twenty-server/src/engine/core-modules/two-factor-method/two-factor-method.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+
+import { Repository } from 'typeorm';
+
+import { TwoFactorMethod } from './two-factor-method.entity';
```

Comment:
general about the naming: shouldn't we use two-factor-authentication instead of two-factor-method which seems more standard?
- for data model: two-factor-method makes sense as a shortcut for two-factor-authentication-method
- but for the module I would name it two-factor-authentication